### ISSUE TITLE
Font does not work in XeTeX

### ISSUE THREAD

**Issue by blmayer** [klepas/open-baskerville#21]
Great work, gorgeous fonts, and specially, thanks for including the long s glyph, its ligatures and ct and st ligatures, no one ever did so. But one problem: Including especial characters or numbers on the font name makes it not work in XeTeX, so could you remove it please? It will be great to see "Open Baskerville" only on the font name. Thanks.

**Comment by codingisacopingstrategy** [comment]
Hello,

Thanks for your report!
But isn’t that a XeTeX bug? Did you file a bug with them already?

Cheers,

Eric

**Comment by blmayer** [comment]
I didn't filed a big yet just searched throughout mailing lists and forums. They say it's a limitation, because TeX interprets the period like the start of the font extension, thus it's not expected to find versions, as they usually have periods, in the font name.

**Comment by rwmpelstilzchen** [comment]
I would like to add there is no reason to include version number as a part of the font’s name. Almost nobody does that, and for a good reason. Version is indicated in the font metadata anyway.

@bleemayer: 

> […] thanks for including the long s glyph, its ligatures and ct and st ligatures, no one ever did so.

You might want to know [Junicode](http://junicode.sourceforge.net/) supports all kinds of ligatures and OpenType features (see [this specimen](http://junicode.sourceforge.net/Junicode.pdf)) and IMHO it looks great.

**Comment by blmayer** [comment]
You're right @rwmpelstilzchen! Thanks for the suggestion, will be taking a look at it.

**Comment by Lilmama93** [comment]
Did you take a look at it?
